consoleApp/src/main/java/com/infoshare/database/FileDb.java

```diff
@@ -0,0 +1,168 @@
+package com.infoshare.database;
+
+import com.infoshare.domain.*;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+public class FileDb implements DB {
+    private static final String VOLUNTEER_DB_FILE_NAME = "Volunteer.csv";
+    private static final String REQUEST_DB_FILE = "NeedRequest.csv";
+    private static final SimpleDateFormat df = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss"); // uzywamy df do formatowania i parsowania dat
+    private Object Volunteer;
+
+
+    public FileDb() {
+    }
+
+    @Override // zapis / aktualizacja danych  wolontariusza
+    public void saveVolunteer(Volunteer volunteer) throws IOException {
+        // sprawdzamy czy istnieje plik do zapisu
+        if (!Files.exists(Paths.get(VOLUNTEER_DB_FILE_NAME))) {
+            try {
+                Files.createFile(Paths.get(VOLUNTEER_DB_FILE_NAME));
+            } catch (IOException e) {
+                e.printStackTrace();
+            }
+        }
+        // Sprawdzamy czy wolontariusz istnieje
+        if (getVolunteer(volunteer.getEmail()) == null) {
+            FileWriter fileWriter = new FileWriter(VOLUNTEER_DB_FILE_NAME, true);
+            fileWriter.write(volunteer.getName() + "," + volunteer.getLocation() + "," + volunteer.getEmail() + ","
+                    + volunteer.getPhone() + "," + volunteer.getTypeOfHelp() + "," + volunteer.isAvailable() + "\n");
```

Comment:
volunteer.getTypeOfHelp() - sprawdź proszę w pliku czy TypeOfHelp jest poprawnie zapisywany, tj. czy jest zapisany cała nazwa wraz z _ (np. WALKING_THE_DOG, a nie WALKING THE DOG)

Reply:
zapisuje jako `WALKING_THE_DOG`